src/sdk/model/org/reports/anti-malware-over-time.ts

```diff
@@ -0,0 +1,49 @@
+/* istanbul ignore next: autogenerated */
```

Comment:
i think this has to be sitting on top of the class declaration (not before the imports). you can verify but i believe i already tried that. comment applies to all classes

Reply:
ah yeah you right, let me update that